### DISCUSSION TITLE
Single pull request causes "something went wrong while executing your query" in pull requests page

### DISCUSSION THREAD

**Discussion by metagn** [community/community#128091] [Mobile]
### Select Topic Area

Bug

### Body

The pull requests page for https://github.com/nim-lang/Nim/ gives an error for me on mobile whenever the pull request https://github.com/nim-lang/Nim/pull/23610 would normally be visible. Searching with specific filters to ignore this pull request shows the pull requests normally.

![Screenshot_2024-06-11-09-21-50-921_com github android-edit](https://github.com/community/community/assets/10591326/6a7114ca-463a-44e3-9d48-86caebeb71f5)

Also, opening the webpage for this pull request on Firefox gives a server error, but I'm currently not able to look at the response data for this. The full pull requests page (https://github.com/nim-lang/Nim/) opens without problems on Firefox however, unlike the mobile app.

![IMG_20240611_092424](https://github.com/community/community/assets/10591326/d65ae030-c2c5-46db-a2d2-c69892ddb962)

**Comment by elisealix22** [comment]
@metagn thank you for reporting this! I created an issue to look into this internally and we will take a look. Please let us know if you have any other trouble in the meantime.
